textworld/envs/wrappers/viewer.py

```diff
@@ -6,7 +6,11 @@
 from typing import Tuple
 
 from textworld.core import Environment, GameState, Wrapper
-from textworld.render.serve import VisualizationService
+
+try:
+    from textworld.render.serve import VisualizationService
+except ModuleNotFoundError:
```

Comment:
Is this not just going to cause a `NameError` later on when we use `VisualizationService`?